data/topic-schema/src/main/java/net/corda/schema/Schemas.java

```diff
@@ -268,13 +268,12 @@ private VirtualNode() {
     public static final class ScheduledTask {
         private ScheduledTask() {}
 
-        // TODO - duplicated to ease migration from first to second. First can be removed once integrated with runtime-os
-        public static final String SCHEDULED_TASK_DB_PROCESSOR = "scheduled.task.db.processor";
         public static final String SCHEDULED_TASK_TOPIC_DB_PROCESSOR = "scheduled.task.db.processor";
         public static final String SCHEDULED_TASK_NAME_DB_PROCESSOR = "deduplication-table-clean-up-task";
-        public static final String SCHEDULED_TASK_MAPPER_PROCESSOR = "scheduled.task.mapper.processor";
+        public static final String SCHEDULED_TASK_TOPIC_MAPPER_PROCESSOR = "scheduled.task.mapper.processor";
+        public static final String SCHEDULED_TASK_NAME_MAPPER_CLEANUP = "flow-mapper-state-cleanup";
         public static final String SCHEDULED_TASK_TOPIC_FLOW_PROCESSOR = "scheduled.task.flow.processor";
-        public static final String SCHEDULED_TASK_NAME_SESSION_TIMEOUT = "flow-session-timout";
+        public static final String SCHEDULED_TASK_NAME_SESSION_TIMEOUT = "flow-session-timeout";
```

Comment:
Any reason for using `-` instead of `.` as the separator within the topic name for these topics?, wouldn't it be better to keep consistency with the others?. Not a blocking concern, just out of curiosity.

Reply:
They're not topics, they're names of scheduled tasks. I think mixing these up is perhaps a little questionable, but not to the extent where I'd want to change it here.

Reply:
👍